mm2src/mm2_net/src/wasm_ws.rs

```diff
@@ -219,7 +218,9 @@ pub fn spawn_ws_transport(idx: ConnIdx, url: &str) -> InitWsResult<(WsOutgoingSe
         // do any action to move the `closures` into this async block to keep it alive until the `state_machine` finishes
         drop(closures);
     };
-    spawn(fut);
+
+    // The state machine will finish almost immediately once `outgoing_tx` is dropped.
+    unsafe { spawn(fut) };
```

Comment:
Is it possible to also use `spawn_abortable` here?

Reply:
Done